src/lib/holocene/combobox/combobox.svelte

```diff
@@ -301,7 +301,9 @@
     </Button>
     {#if $$slots.action}
       <div class="ml-1 h-full w-1 border-l-2 border-subtle" />
-      <slot name="action" />
+      <div class="-ml-px px-0.5">
```

Comment:
I'm not sure I love this. Can we just add `px` here?

Reply:
Refactored [here](https://github.com/temporalio/ui/pull/2214/commits/816924bd2c8a5b4efc23b2b325b40c80dc59ecd5) ✅